sql/core/src/main/java/org/apache/spark/sql/expressions/javalang/typed.java

```diff
@@ -74,4 +71,40 @@
   public static <T> TypedColumn<T, Long> sumLong(MapFunction<T, Long> f) {
     return new TypedSumLong<T>(f).toColumnJava();
   }
+
+  /**
+   * Min aggregate function for floating point (double) type.
+   *
+   * @since 3.0.0
+   */
+  public static <T> TypedColumn<T, Double> min(MapFunction<T, Double> f) {
+    return new JavaTypedMinDouble<T>(f).toColumn();
+  }
+
+  /**
+   * Min aggregate function for integral (long, i.e. 64 bit integer) type.
+   *
+   * @since 3.0.0
+   */
+  public static <T> TypedColumn<T, Long> minLong(MapFunction<T, Long> f) {
+    return new JavaTypedMinLong<T>(f).toColumn();
+  }
+
+  /**
+   * Min aggregate function for floating point (double) type.
```

Comment:
`Min` -> `Max`. Actually I will elaborate it more. Please avoid abbreviation like min or max in doc.

Reply:
Ok,thanks